executor/common_linux.h

```diff
@@ -1557,6 +1557,75 @@ static long syz_io_uring_submit(volatile long a0, volatile long a1, volatile lon
 
 #endif
 
+#if SYZ_EXECUTOR || __NR_syz_usbip_server_init
+
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+
+static long syz_usbip_server_init(void)
+{
+	int fd;
+	int rc;
+	int socket_pair[2];
+	char buffer[100];
+	int server_fd, client_fd;
+
+	rc = socketpair(AF_UNIX, SOCK_STREAM, 0, socket_pair);
+	if (rc < 0) {
+		fail("syz_usbip_server_init : socketpair failed: %d\n", rc);
+	}
+
+	client_fd = socket_pair[0];
+	server_fd = socket_pair[1];
+	sprintf(buffer, "%s %d %s", "0", client_fd, "65537 3");
+
+	fd = open("/sys/devices/platform/vhci_hcd.0/attach", O_WRONLY | O_CREAT | O_TRUNC, S_IRWXU);
```

Comment:
As discussed we need one HCD per proc via `procid`.

Reply:
We sure don't want to create it here :)
Drop `| O_CREAT | O_TRUNC, S_IRWXU` part.